cl/lib/types.py

```diff
@@ -1,12 +1,22 @@
 from typing import Any, Dict, List, TypedDict, Union
 
 from django.http import HttpRequest
+from eyecite.models import (
+    FullCaseCitation,
+    IdCitation,
+    ShortCaseCitation,
+    SupraCitation,
+)
 
 from cl.users.models import User
 
 CleanData = Dict[str, Any]
 TaskData = Dict[str, Any]
 
+SupportedCitationType = Union[
```

Comment:
The idea here is that the next version of eyecite that is cut (i.e., after the current 2.1 tag) will have a bunch of new citation types (journals, statutes, etc.). CL may not support them all at once, so I thought it was prudent to just create a type shorthand to signify which citations we're dealing with right now (rather than just assuming we can handle all `CitationBase` objects).